pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -172,6 +183,24 @@ func ReconcileDevWorkspace(deployContext *deploy.DeployContext) (bool, error) {
 	return true, nil
 }
 
+func isDevWorkspaceDeploymentExists(deployContext *deploy.DeployContext) bool {
+	var DevWorkspaceNamespaces = []string{DevWorkspaceNamespace, "openshift-operators"}
```

Comment:
It is enough to check in DevWorkspaceNamespace since it is the only namespace where che-operator creates it manually

Reply:
It is better to check in openshift-operators if somebody deploys the dwo manually from OperatorHub and deploy che from stable Channel